### PR TITLE
Releace v 0.1.34

### PR DESCRIPTION
**Pull request by Anubhvv**
Release Version 0.1.34:

What's New:

Fluent V2:

```
Fix/Enhancement:
                1. Persona chip Bug Fixed: Persona height will change when the system font size is increased
		2. Improved the visibility of the ToolTip Activity Caret in dark mode
		3 Fixed the gap between the Drawer and the Navigation bar
		4. Made the SectionHeader non-clickable when no onClick or null onClick is passed
		5. Minor enhancement in demo app
		6. Readme has been updated to include information about the v2 components as well.
```

### CODE REVIEW

File: FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/Demos.kt [hunk 1]
```diff
@@ -167,20 +167,20 @@ val V2DEMO = arrayListOf(
...
-    Demo(V2CARD_NUDGE, V2CardNudgeActivity::class, Badge.Modified),
+    Demo(V2CARD_NUDGE, V2CardNudgeActivity::class),
     Demo(V2CITATION, V2CitationActivity::class),
     Demo(V2CONTEXTUAL_COMMAND_BAR, V2ContextualCommandBarActivity::class),
     Demo(V2DIALOG, V2DialogActivity::class),
-    Demo(V2DRAWER, V2DrawerActivity::class),
+    Demo(V2DRAWER, V2DrawerActivity::class, Badge.Modified),
     Demo(V2LABEL, V2LabelActivity::class),
-    Demo(V2LIST_ITEM, V2ListItemActivity::class),
+    Demo(V2LIST_ITEM, V2ListItemActivity::class, Badge.APIBreak),
```
Comment (praveen970) [inline FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/Demos.kt:179]: shouldnt it be modified

File: FluentUI.Demo/src/main/assets/dogfood-release-notes.txt [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 What's New:
 
 Fluent V2:
 	Fix/Enhancement:
-		1. Added Functionality to the BottomSheet with TestCases: Enable/Disable Swipe Down Dismiss with more intuitive accessibility controls
-		2. ToolTip Alignment Bug Fixed: Resolved an issue where ToolTips were not aligning correctly, ensuring a visually consistent and user-friendly interface
-		3. CardNudge Enhancement: Fixed a right-to-left (RTL) issue affecting swiping left and right within CardNudge, ensuring a smoother interaction
-		4. Vertical Lists Icon Color Consistency: Addressed an issue where disabled icons in vertical lists had inconsistent colors. Icons now maintain uniform appearance for improved visual clarity
+	    1. Persona chip Bug Fixed: Persona height will change when the system font size is increased
+		2. Improved the visibility of the ToolTip Activity Caret in dark mode
```
Comment (praveen970) [inline FluentUI.Demo/src/main/assets/dogfood-release-notes.txt:8]: Can you check the formatting

File: FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/Demos.kt (hunk 1 above)
Comment (Anubhvv) [reply FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/Demos.kt:179]: API has changed, before this there was no parameter for onClick, now it has and also default behavior has changed

File: FluentUI.Demo/src/main/assets/dogfood-release-notes.txt (hunk 2 above)
Comment (Anubhvv) [reply FluentUI.Demo/src/main/assets/dogfood-release-notes.txt:8]: image
Seems okay in local IDE

**Review by praveen970 — APPROVED** [review]

**Pull request merged**